src/ext/index.js

```diff
@@ -34,7 +34,15 @@ export default function ext(env) {
       exploration: true,
       cssScaling: true,
     },
-    importProperties,
-    exportProperties,
+    importProperties(exportFormat, initialProperties, extension, hypercubePath) {
+      const defaultPropertyValues = {
+        defaultDimension: extension.getDefaultDimensionProperties(),
+        defaultMeasure: extension.getDefaultMeasureProperties(),
+      };
+      return importProperties({ exportFormat, initialProperties, extension, hypercubePath, defaultPropertyValues });
```

Comment:
I'm looking at the `importProperties` call in the `convertTo` method in nebula. The parameters don't really match. Is it a different `importProperties` being called in nebula?